### DISCUSSION TITLE
Account marked as spammy and went invisible

### DISCUSSION THREAD

**Discussion by vladimir1257** [community/community#110990] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hey GitHub Community,

I'm here because I'm facing some tough challenges and really could use your help. On February 22, I put in a Reinstatement Request after running into an issue with the [Atlantis server](https://github.com/runatlantis/atlantis). Despite setting up my personal token for a higher rate limit, I'm still stuck at the default 60 requests per hour, according to the GitHub REST API [docs](https://docs.github.com/en/rest/using-the-rest-api/rate-limits-for-the-rest-api?apiVersion=2022-11-28). As a result, GitHub marks my account as Spammy and turns it into a real ghost!

The odd part is, if I hit the limit of 60 requests as an unauthorized user, why does GitHub block `my` user account instead of my `IP`? It seems like a more nuanced approach to API rate limiting could avoid such disruptive outcomes.

To make things worse, my account suddenly went "invisible" to all my colleagues. I can't make any PRs since they just don't show up, I can't ask anything in the community, and nobody can even see my public profile. It's pretty disruptive to our work and honestly quite frustrating.

I've got GitHub Pro and Copilot, and it's perplexing that I'm hitting these barriers. I'm hoping someone here might have gone through something similar or knows a workaround. I have already opened a Reinstatement Request but they do not seem to care to respond.

Here's what I'm looking for:

1. Has anyone else had their account go missing in action like this? What did you do to get back to normal cause the GitHub support center doesn’t answer themself nor do they allow me to post a question to this community?
2. Tips on getting a faster response to a Reinstatement Request, or if there's something else I should be doing to sort this out?
3. Any idea why the API rate limit seems locked at 60 requests per hour for my personal token? How can I fix this?
4. Given that I'm hitting this rate limit, shouldn't the restrictions be based on IP rather than disabling user account access, especially considering the token is configured with specific repo access?

Any advice or shared experiences would be incredibly valuable right now. Thanks so much for taking the time to read and help out.

Cheers,
Ex @Vladimir-125

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation

Acces the [Github Support](https://support.github.com/)

Click on My Ticket - New Ticket - And Finally describe it

Good luck!

**Comment by Lanrad23** [comment]
I think a similar issue has been discussed already. Check this thread https://github.com/orgs/community/discussions/67196. I think you might find some helpful points from that discussion.

**Comment by vladimir1257** [comment]
@gil-son, @Lanrad23 Thanks for your answers, but as I told you already, I have opened a `Reinstatement Request` ticket and explained the problem in detail. I do not see any actions despite the third week has started.

**Comment by Lanrad23** [comment]
Well, as far as I know, GitHub might block user accounts instead of IP addresses to directly stop the actions they think are harmful, without impacting other users who might be using the same IP. This method can sometimes cause issues like what you're experiencing.

**Comment by vladimir1257** [comment]
Hey, how long does it usually takes to resolve an issue? It has been 3 weeks, and I did not get any response yet. I am so frustrated and disappointed with the GitHub support team for such a slow resolution of a simple problem!

**Comment by therealsupermax** [comment]
Has anyone solved this issue yet? I'm still having trouble fixing my issue here

**Comment by SalahAlDin2021** [comment]
Hello, Are the Github Support reply to the message ?
And how much time need to reply to my message and fix the issue ?
@vladimir1257 are they fix your problem, and how much takes ?

**Comment by SalahAlDin2001** [comment]
Hello, Are the Github Support reply to the message ?
And how much time need to reply to my message and fix the issue ?
@vladimir1257 are they fix your problem, and how much takes ?

**Comment by vrj93** [comment]
Mine too flagged for unacknowledged reasons. Github support is not actually supportive.

**Comment by edwardfernandes** [comment]
I opened a ticket on February 3rd and so far I haven't had any response and being marked as "spammy" is already affecting my workflow too much. I need a solution, urgently.

https://support.github.com/ticket/personal/0/3213804

**Comment by Hassanawi** [comment]
my account is marked spammy and created ticket #3420408 . i created two tickets earlier but was not replied.  kindly remove spammy lable from my account 

https://support.github.com/ticket/personal/0/3420408
